src/main/scala/cromwell/binding/values/WdlFloatValue.scala

```diff
@@ -0,0 +1,28 @@
+package cromwell.binding.values
+
+import cromwell.binding.types.WdlFloatType
+
+case class WdlFloatValue(value: Float) extends WdlPrimitiveValue {
+  val wdlType = WdlFloatType
+  def add(rhs: WdlValue): WdlValue = {
+    rhs match {
+      case r:WdlFloatValue => WdlFloatValue(value + r.value)
```

Comment:
Note that if anything else comes in you'll get a match exception. Can something else come in? Is it just not yet done at the moment?